src/core/server/saved_objects/service/lib/search_dsl/query_params.test.ts

```diff
@@ -625,6 +625,27 @@ describe('#getQueryParams', () => {
         ]);
       });
     });
+
+    describe('when using workspace search', () => {
```

Comment:
Can you add cases for searching multiple workspaces? We need to test the **OR** logic among workspaces.

Reply:
Can add later when we start supporting share case, currently we should only have one workspace.